lib/task_helpers/imports/roles.rb

```diff
@@ -0,0 +1,33 @@
+module TaskHelpers
+  class Imports
+    class Roles
+      def import(options = {})
+        return unless options[:source]
+
+        available_features = MiqProductFeature.all
```

Comment:
@branic This is good but I think you should move it into the method that needs it instead of passing it as an argument. You can just put it right above line 23 below, just above the `roles.each do |r|` line.